workspaces/adr/.changeset/healthy-horses-wait.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage-community/plugin-adr-backend': patch
+---
+
+Deprecated the `search` types, which should be now imported from "@backstage-community/search-backend-module-adr" instead
```

Comment:
```suggestion
The `DefaultAdrCollatorFactory` has been deprecated in favour of using the new `@backstage-community/search-backend-module-adr` module.
```

Reply:
Reworded the changelog. Also added a second paragraph with the types that have been moved